packages/holocron-dev-server/__tests__/src/utils/rendering.spec.js

```diff
@@ -0,0 +1,313 @@
+  test('generates script meta data for modules to be rendered as script tags', () => {
+    const rootModule = true;
+    const name = 'root-module';
+    const moduleName = name;
+    const modulePath = '/path/to/module';
+    const src = '/static/modules/root-module/root-module.js';
+    const modules = [
+      //  only the root module
+      {
+        rootModule, moduleName, modulePath, name, src,
+      },
+    ];
+    const moduleMap = {
+      modules: {
+        [moduleName]: {
+          browser: {
+            url: src,
+          },
+        },
+      },
+    };
+    const webpackScriptsResult = [
+      {
+        name: 'runtime',
+        src: '/static/modules/runtime/runtime.js',
+      },
+      {
+        name,
+        src,
+      },
+    ];
+    expect(
+      getWebpackScriptsForLocalModules({
+        modules,
+        moduleMap,
+        rootModuleName: moduleName,
+      })
+    ).toEqual(webpackScriptsResult);
+    expect(
+      getWebpackScriptsForLocalModules({
+        moduleMap,
+        rootModuleName: moduleName,
+      })
+    ).toEqual(webpackScriptsResult);
+  });
+
+  test('returns runtime and root module scripts if no modules supplied', () => {
+    const name = 'root-module';
+    const src = '/static/modules/root-module/root-module.js';
+    const modules = [];
+    const moduleMap = {
+      modules: {
+        [name]: {
+          browser: {
+            url: src,
+          },
+        },
+      },
+    };
+    const webpackScripts = getWebpackScriptsForLocalModules({
+      modules,
+      moduleMap,
+      rootModuleName: name,
+    });
+    expect(webpackScripts).toEqual([
+      {
+        name: 'runtime',
+        src: '/static/modules/runtime/runtime.js',
+      },
+      {
+        name,
+        src,
+      },
+    ]);
+  });
+
+  test('orders the scripts correctly', () => {
+    const rootModule = true;
+    const name = 'root-module';
+    const moduleName = name;
+    const modulePath = '/path/to/module';
+    const src = '/static/modules/root-module/root-module.js';
+    const modules = [
+      {
+        name: 'child-module',
+        src: 'child-module',
+      },
+      {
+        name: 'another-child-module',
+        src: 'another-child-module',
+      },
+      {
+        name: 'yet-another-child-module',
+        src: 'yet-another-child-module',
+      },
+      {
+        rootModule, moduleName, modulePath, name, src,
+      },
+      {
+        name: 'last-child-module',
+        src: 'last-child-module',
+      },
+    ];
+    const moduleMap = {
+      modules: {
+        [moduleName]: {
+          browser: {
+            url: src,
+          },
+        },
+      },
+    };
+    const webpackScripts = getWebpackScriptsForLocalModules({
+      modules,
+      moduleMap,
+      rootModuleName: moduleName,
+    });
+    expect(webpackScripts).toEqual([
+      {
+        name: 'runtime',
+        src: '/static/modules/runtime/runtime.js',
+      },
+      {
+        name,
```

Comment:
just for clarity
```suggestion
        name: 'root-module',
```

or update the const to be `rootModuleName`